src/old/lib/components/Chips/ChipsList.tsx

```diff
@@ -9,11 +9,7 @@ export interface IChipsListProps {
   max?: boolean;
 }
 
-const ChipsList: React.FC<IChipsListProps> = ({
-  checkedNames,
-  isLabelItem,
-  max,
-}) => {
+const ChipsList: React.FC<IChipsListProps> = ({ checkedNames }) => {
```

Comment:
since we don't use `max` and `isLabelItem` anymore, may it make sense to remove them from `IChipsListProps` as well?

Reply:
I think its  better to move `IChipsListProps` to a new file, cause another components use this interfece